main.py

```diff
@@ -37,43 +36,64 @@ def request_github_internship24_data() -> List[Opportunity]:
     """Scrapes Internship Data '24 from Github Repo"""
 
     url = os.getenv("GITHUB_INTERN24_URL")
-
     parse_content = utils.content_parser(url)
-
-    table_div = parse_content.find_all("tr")
     github_list = []
-    for table in table_div:
-        cell = table.find_all("td")
-        if len(cell) == 3:
-            _company = cell[0].find("a")
-            _titles = cell[2].find_all(
-                "a"
-            )  # Create a List[str] which contain all the titles
-            _location = cell[1].text
-
-            if (
-                (_company and "href" in _company.attrs)
-                or any(title and "href" in title.attrs for title in _titles)
-                # If the current row has any job titles that contain a link attribute
-                and "Closed" not in _titles
-            ):
-                _company_link = (
-                    _company["href"] if _company and len(_titles) == 1 else ""
-                )  # If the length of the current title(s) is 1 then we grab the company link immediately
-                _company_name = (
-                    _company.text.strip() if _company else cell[0].text.strip()
+    td_elems = parse_content.find_all("td")
+
+    # A temporary list holding the current rows internship data
+    temp = []
+
+    for cell in td_elems:
+        temp.append(cell)
+
+        if (
+            len(temp) == 3
+        ):  # A length of three indicates a complete row has been searched
+            company = temp[0]
+            location = temp[1].text
+            title = temp[2]
+            t = temp[2].find_all("a")
+
+            if len(title) == 1:
+                """
+                If the title length consists of one element,
+                this indicates that the link exists within
+                the company title and the company title only.
+                """
+                opportunity = Opportunity(
+                    company.text,
+                    title.text,
+                    location,
+                    company.find("a")["href"],
+                    0,
+                    OpportunityType.INTERNSHIP.value,
                 )
+                github_list.append(opportunity)
+
+            for i in t:
```

Comment:
Not a docstring

[Reference](https://peps.python.org/pep-0257/#what-is-a-docstring)

Reply:
Implemented [here](https://github.com/acmcsufoss/acmOpportunities/pull/11/commits/1084fabdbf39ec53e031f7dd4998f397ca913f72)

Reply:
> Not a docstring (line 74-82)
> 
> [Reference](https://peps.python.org/pep-0257/#what-is-a-docstring)

Drag your mouse across the lines you intend to make your suggestion on.

![image](https://github.com/acmcsufoss/acmOpportunities/assets/31261035/d1b9e818-c266-4f99-ad92-3158b079eb9e)